lib/Doctrine/Migrations/Tools/Console/Command/DiffCommand.php

```diff
@@ -81,6 +82,12 @@ protected function configure() : void
                 InputOption::VALUE_OPTIONAL,
                 'Check Database Platform to the generated code.',
                 true
+            )
+            ->addOption(
+                'allow-empty-diff',
+                null,
+                InputOption::VALUE_NONE,
+                'Do not throw an exception when no changes are detected.'
```

Comment:
```suggestion
                'Don\'t throw an exception when no changes are detected.'
```

Reply:
It wasn't my idea. Please see https://github.com/doctrine/migrations/pull/840#discussion_r307963742